.github/workflows/verify-binary-installation.yml

```diff
@@ -0,0 +1,119 @@
+name: 'Install Dashboards with Plugin via Binary'
+
+on: [push, pull_request]
+env:
+  OPENSEARCH_VERSION: '3.0.0'
```

Comment:
No, the "branch-switch-if-possible" step would take care of this. We are just setting 3.0.0 as a default, since 'main' is not a number. I'm leaning towards just leaving this as is, since this would be consistent with other workflows, since I copied it from them.